### PR TITLE
EKS Integration doc updates

### LINKED ISSUE #816: [Enhancement] Update and generalize EKS installation instructions
### Description

On the [EKS installation page](https://docs.kubecost.com/install-and-configure/install/provider-installations/aws-eks-cost-monitoring), there are things that are no longer accurate with respect to overriding values supplied to the `helm install` commands. As a better practice, we should generalize these installation instructions and avoid providing precise commands and all their inputs and values and instead defer users to the canonical location where these can be found: in the artifact repositories themselves. The values files will (or should) have this information. The documentation should just provide a general overview and otherwise direct users there.

### LINKED ISSUE #1030: [Bug] deployment name is wrong
### Page link

https://docs.kubecost.com/install-and-configure/install/provider-installations/aws-eks-cost-monitoring

### Description

deployment name is wrong

[wrong]
```
kubectl port-forward --namespace kubecost deployment/kubecost-cost-analyzer 9090
```

[correct]
```
kubectl port-forward --namespace kubecost deployment/cost-analyzer 9090
```
### Expected behavior

The correct deployment name must be written.

### PR DESCRIPTION
**Pull request by bstuder99**
Closes #816
Closes #1030 

Makes updates to EKS Integration doc, removes mentions to /develop/ branch of Helm chart repo, and links from a newly-made eks-v2.2 branch to ensure the content remains static and we can track if future changes may impact this content. Updates the deployment name in port-forward commands.

### CODE REVIEW

**Comment by bstuder99** [comment]
Going to update this with #1028 also

File: install-and-configure/install/provider-installations/aws-eks-cost-monitoring.md [hunk 1]
```diff
@@ -168,7 +168,7 @@ In your environment, run the following command from your terminal to install Kub
 helm upgrade -i kubecost \
 oci://public.ecr.aws/kubecost/cost-analyzer --version <$VERSION> \
 --namespace kubecost --create-namespace \
--f https://raw.githubusercontent.com/kubecost/cost-analyzer-helm-chart/develop/cost-analyzer/values-eks-cost-monitoring.yaml
+-f https://raw.githubusercontent.com/kubecost/cost-analyzer-helm-chart/eks-v2.2/cost-analyzer/values-eks-cost-monitoring.yaml
```
Comment (chipzoller) [inline install-and-configure/install/provider-installations/aws-eks-cost-monitoring.md:171]: This isn't a valid branch name. Don't hard-code anything in here, just use something like `<$VERSION>` like above.

File: install-and-configure/install/provider-installations/aws-eks-cost-monitoring.md [hunk 2]
```diff
@@ -179,9 +179,7 @@ To install Kubecost on Amazon EKS cluster on AWS Graviton2 (ARM-based processor)
 helm upgrade -i kubecost \
 oci://public.ecr.aws/kubecost/cost-analyzer --version <$VERSION> \
 --namespace kubecost --create-namespace \
--f https://raw.githubusercontent.com/kubecost/cost-analyzer-helm-chart/develop/cost-analyzer/values-eks-cost-monitoring.yaml \
---set prometheus.configmapReload.prometheus.image.repository=jimmidyson/configmap-reload \
---set prometheus.configmapReload.prometheus.image.tag=v0.7.1
+-f https://raw.githubusercontent.com/kubecost/cost-analyzer-helm-chart/eks-v2.2/cost-analyzer/values-eks-cost-monitoring.yaml \
```
Comment (chipzoller) [inline install-and-configure/install/provider-installations/aws-eks-cost-monitoring.md:182]: Ditto

File: install-and-configure/install/provider-installations/aws-eks-cost-monitoring.md [hunk 3]
```diff
@@ -236,7 +234,7 @@ In your environment, run the following command from your terminal to install Kub
 helm upgrade -i kubecost \
 oci://public.ecr.aws/kubecost/cost-analyzer --version <$VERSION> \
 --namespace kubecost --create-namespace \
--f https://raw.githubusercontent.com/kubecost/cost-analyzer-helm-chart/develop/cost-analyzer/values-eks-cost-monitoring.yaml
+-f https://raw.githubusercontent.com/kubecost/cost-analyzer-helm-chart/eks-v2.2/cost-analyzer/values-eks-cost-monitoring.yaml
```
Comment (chipzoller) [inline install-and-configure/install/provider-installations/aws-eks-cost-monitoring.md:237]: Ditto

File: install-and-configure/install/provider-installations/aws-eks-cost-monitoring.md [hunk 4]
```diff
@@ -247,9 +245,7 @@ To install Kubecost on an EKS-A cluster on AWS Graviton2 (ARM-based processor),
 helm upgrade -i kubecost \
 oci://public.ecr.aws/kubecost/cost-analyzer --version <$VERSION> \
 --namespace kubecost --create-namespace \
--f https://raw.githubusercontent.com/kubecost/cost-analyzer-helm-chart/develop/cost-analyzer/values-eks-cost-monitoring.yaml \
---set prometheus.configmapReload.prometheus.image.repository=jimmidyson/configmap-reload \
---set prometheus.configmapReload.prometheus.image.tag=v0.7.1
+-f https://raw.githubusercontent.com/kubecost/cost-analyzer-helm-chart/eks-v2.2/cost-analyzer/values-eks-cost-monitoring.yaml \
```
Comment (chipzoller) [inline install-and-configure/install/provider-installations/aws-eks-cost-monitoring.md:248]: Ditto

**Review by chipzoller — CHANGES_REQUESTED** [review]

File: install-and-configure/install/provider-installations/aws-eks-cost-monitoring.md (hunk 1 above)
Comment (bstuder99) [reply install-and-configure/install/provider-installations/aws-eks-cost-monitoring.md:171]: GitHub allowed me to create a branch named this, and the link seems to work, but I can just create a new one `2.2` or something like that, is that better?

Comment (chipzoller) [reply install-and-configure/install/provider-installations/aws-eks-cost-monitoring.md:171]: Wait, why are you creating new branches in the Helm chart repository? You don't need to do that, just use the variable so the users know they need to complete it on their end.

**Review by chipzoller — COMMENTED** [review]
Rather than "applicable" could I suggest a wording like "Am I eligible for the Kubecost EKS optimized license?" Or "Does my installation qualify for an EKS optimized license?"

Also, did you create the `eks-v2.2` branch in the Helm chart repo? If so, that needs to be removed.

I also see some conversations here which aren't resolved.

**Review by chipzoller — APPROVED** [review]

**Pull request merged**